rclpy/rclpy/utilities.py

```diff
@@ -44,3 +47,23 @@ def get_rmw_implementation_identifier():
     # imported locally to avoid loading extensions on module import
     from rclpy.impl.implementation_singleton import rclpy_implementation
     return rclpy_implementation.rclpy_get_rmw_implementation_identifier()
+
+
+def timeout_sec_to_nsec(timeout_sec):
+    """
+    Convert timeout in seconds to rcl compatible timeout in nanoseconds.
+
+    Python tends to use floating point numbers in seconds for timeouts. This utility converts a
+    python-style timeout to an integer in nanoseconds that can be used by rcl_wait.
+
+    :param timeout_sec: Seconds to wait. Block forever if None or negative. Don't wait if < 1ns
```

Comment:
Nit: `Don't wait if 0 <= timeout_sec < 1ns`
maybe it's clear enough as is

Reply:
If it is okay with you, I would like to leave this as is to avoid wrapping the line.

Reply:
:+1: works for me